src/averageAges.js

```diff
@@ -55,7 +67,23 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = people.filter(person =>
+    !onlyWithSon
```

Comment:
It is better to use '&&' instead of nested ternary operators for better readability.